docs/src/quickstart.md

```diff
@@ -72,29 +111,49 @@ smoothly with the vast majority of functions in JuliaImages.
 
 ## Array elements are pixels (and vice versa)
 
-In some languages, an RGB image is represented as an `m × n × 3` array,
-where the third dimension is the color-channel dimension.
-In such cases the representation of a single pixel is spread out among
-3 array elements.
-In JuliaImages, the typical representation of such an image would be
-an `m × n` array of `RGB` values.
-Consequently, every element of the array corresponds to one pixel,
-and conversely each pixel is represented by exactly one array element:
+```@setup pixel
+using Images, MosaicViews
+```
+
+Elements of image are called **pixels**; in JuliaImages we provide an
+abstraction on this concept.
 
-```jldoctest
-using Images, TestImages
+For example, we have `Gray` for grayscale image, `RGB` for RGB image, `Lab`
+for Lab image, and etc.
 
-img = testimage("mandrill")
-img[350,225]    # pick a reddish pixel from the nose of the mandrill
+Creating a pixel is initializing a struct of that type:
 
-# output
+```@example pixel
+Gray(0.0) # black
+Gray(1.0) # white
+RGB(1.0, 0.0, 0.0) # red
+RGB(0.0, 1.0, 0.0) # green
+RGB(0.0, 0.0, 1.0) # blue
+[RGB.(Gray(0.0)) RGB.(Gray(1.0)) RGB(1.0, 0.0, 0.0) RGB(0.0, 1.0, 0.0) RGB(0.0, 0.0, 1.0)] # hide
+```
+
+and image is just an array of pixel objects:
 
-RGB{N0f8}(0.937,0.294,0.231)
+```@repl pixel
+img_gray = rand(Gray, 2, 2)
+img_rgb = rand(RGB, 2, 2)
+img_lab = rand(Lab, 2, 2)
+```
+```@example pixel
+mosaicview(cat(RGB.(img_gray), # hide
+               RGB.(img_rgb), # hide
+               RGB.(img_lab), dims=3), # hide
+           RGB(1, 1, 1), # hide
+           nrow=1, npad=2) # hide
 ```
 
-(In ImageView, you can "hover" over pixels and see their coordinates in the statusbar.)
-The meaning of `N0f8` is summarized [below](@ref The-0-to-1-intensity-scale) and described in detail
-[later](@ref fixedpoint).
+As you can see, both `img_rgb` and `img_lab` images are of size
+``2 \times 2`` instead of ``2 \times 2 \times 3`` or ``3 \times 2 \times 2``;
+RGB image has `RGB` pixels, Lab image has `Lab` pixel.
+
+!!! note
+    It's recommended to use `Gray` compared to `Number` in JuliaImages since it has a
```

Comment:
It's recommended to use `Gray` instead of the `Number` type in JuliaImages since it signals that the array of numbers is best interpreted as a grayscale image. For example, it triggers `Atom/Juno` and `Jupyter`  to display the array as an image instead of a matrix of numbers. There's no performance overhead for using `Gray` over `Number`.